crates/iota-core/src/authority/authority_per_epoch_store.rs

```diff
@@ -675,6 +677,10 @@ pub struct AuthorityEpochTables {
 
     /// Holds the timestamp of the most recently generated round of randomness.
     pub(crate) randomness_last_round_timestamp: DBMap<u64, CommitTimestampMs>,
+
+    /// Accumulated per-object debts for congestion control.
+    pub(crate) congestion_control_object_debts: DBMap<ObjectID, CongestionPerObjectDebt>,
+    pub(crate) congestion_control_randomness_object_debts: DBMap<ObjectID, CongestionPerObjectDebt>,
```

Comment:
```suggestion
    congestion_control_object_debts: DBMap<ObjectID, CongestionPerObjectDebt>,
    
    /// Accumulated per-object debts for randomness congestion control.
    congestion_control_randomness_object_debts: DBMap<ObjectID, CongestionPerObjectDebt>,
```

Few points:

- both fields can be kept private;
- the second field should have doc string as well.